config-scripts/cups-common.m4

```diff
@@ -448,6 +450,22 @@ AC_DEFINE_UNQUOTED([CUPS_DEFAULT_SYSTEM_AUTHKEY], ["$CUPS_DEFAULT_SYSTEM_AUTHKEY
 AC_SUBST([CUPS_SYSTEM_AUTHKEY])
 AC_SUBST([INSTALLXPC])
 
+dnl Check if monotonic clock is available
+AC_MSG_CHECKING([for monotonic clock])
+AC_LINK_IFELSE([AC_LANG_PROGRAM([
```

Comment:
It should be enough to check that `CLOCK_MONOTONIC` is defined. CUPS requires POSIX compliance.